app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -0,0 +1,31 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class ResponseRedacter
+        ALLOWED_RESPONSE_FIELDS = %w[
+          fraudpoint.score
+          request_id
+          request_result
+          review_status
+          risk_rating
+          summary_risk_score
+          tmx_risk_rating
+        ]
+
+        def self.whilelist_response_and_redact_unwhitelisted_fields(response_json)
```

Comment:
1. let's shorten this name
2. I expected `response_json` to be parsed into a structure already? If the incoming value is a string, I feel like the name `body` might be clearer? And definitely a type annotation would help clarify

```suggestion
        # @param [String] body
        def self.redact(body)
```